tests/Feature/Api/FeedApiTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Package;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class FeedApiTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function it_counts_packages()
+    {
+        Package::factory(5)->create();
+
+        $response = $this->getJson('/api/packages.json');
+
+        $response
+            ->assertStatus(200)
+            ->assertJsonCount(5);
```

Comment:
Can you add an assertion to ensure the structure of the response? I forget what the assertion is off of the top of my head but I want to make sure future changes don't break the contract of this response (ie `author` isn't suddenly removed from the response and breaks people's integrations)

Reply:
Updated the test to assert the JSON structure & updated the name to better reflect what it does. Also added the `/** @test */` DocBlock causing the test not to run in the first place.

Reply:
It might also be worth mentioning it doesn't seem other API tests assert HTTP response code and such, this test is a bit "rogue" from what the other tests cover. 👍

Reply:
> Also added the /** @test */ DocBlock causing the test not to run in the first place.

Great catch!

I appreciate you pointing out how this test differs from existing tests. Asserting the status code is a good practice and other tests can be updated to include that assertion in the future 👍🏾